Automate 'Start a snapshot build' step in beam release guide

boyuanzz opened the pull request:
Automate [Start a snapshot build](https://beam.apache.org/contribute/release-guide/#start-a-snapshot-build)

Note:
It's supposed to trigger the jenkins job but it sometimes works sometimes not. So it needs release manager to check whether build job is been triggered, if not, then add a comment manually.

r: @aaltay @pabloem

aaltay commented on release/src/main/scripts/start_snapshot_build.sh:
```diff
@@ -0,0 +1,85 @@
+
+Run Gradle Publish"
+
+echo "NOTE: If there is no jenkins jib started, please comment generated PR with Run Gradle Publish"
```
`jib` -> `job`

aaltay commented on release/src/main/scripts/start_snapshot_build.sh:
```diff
@@ -0,0 +1,85 @@
+cd ~
+rm -rf ${LOCAL_BEAM_DIR}
+echo "Things remained you need to do manually after build successful:"
+echo "1. Close this generated PR"
```
Could you add copy-pasteable commands here for doing 1 and 2.

boyuanzz commented:
Re @aaltay 
Addressed your comments in commit https://github.com/apache/beam/pull/6042/commits/21b075d11f1a298a0bcf0d6706cf43323ccc9458.
For remained manually operations, it's much easier and straightforward to do all actions in github website.
